src/dotnet-scaffolding/Microsoft.DotNet.Scaffolding.Helpers/ScaffolderBuilderExtensions.cs

```diff
@@ -0,0 +1,116 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+using Microsoft.DotNet.Scaffolding.Helpers.General;
+using Microsoft.DotNet.Scaffolding.Helpers.Steps;
+using Microsoft.DotNet.Scaffolding.Helpers.Templates.DbContext;
+using Microsoft.DotNet.Scaffolding.TextTemplating;
+
+namespace Microsoft.DotNet.Scaffolding.Core.Builder;
+
+internal static class ScaffolderBuilderExtensions
+{
+    /// <summary>
+    /// Adds a new DbContext class 
+    /// </summary>
+    /// <param name="builder"></param>
+    /// <returns></returns>
+    public static IScaffoldBuilder WithDbContextStep(
+        this IScaffoldBuilder builder,
+        DbContextProperties? dbContextProperties = null)
```

Comment:
`dbContextProperties` doesn't appear to be passed in anymore in any of the cases where this is currently called. Can it be removed as a parameter? Or do you know for sure it'll be used in future uses?

Reply:
yep removed, we can iterate on this later. I think I try to anticipate too much flexibility on what other devs will want.